package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Magnet",
+  "name": "project-magnet",
```

Comment:
`react-native` uses this string to configure the some of the Android and iOS applications. So we should make sure that it's valid in the contexts it may be used during a `react-native upgrade`.